FHIR-28548: Add examples instances to IG for "extended example 1"

masnick opened the pull request:
**[FHIR-28548](https://jira.hl7.org/browse/FHIR-28548)**

This PR adds an "extended example" to the IG. This affects `example.html` and the artifacts list, as well as creating the separate pages for all the example instances.

The examples generated by `ExtendedExample1.fsh`.

markkramerus commented:
[MaxPersonaReview.docx](https://github.com/HL7/fhir-mCODE-ig/files/5171118/MaxPersonaReview.docx)

masnick commented:
Responding to [Mark's code review](https://github.com/HL7/fhir-mCODE-ig/files/5171118/MaxPersonaReview.docx):

### General
- [x] In FSH, it is not necessary to write Reference(Patient/example1-mcode-cancer-patient). Reference(example-1-mcode-cancer-patient) suffices.
- [x] For comorbid conditions, you do not need to specify the condition, because it is inherited from the profile
- [x] Likewise for ECOG, you don’t have to have the line * code = LNC#89247-1 "ECOG Performance Status score"
- [x] Make sure to use `* condition.onsetAge = UCUM#a "yr"` rather than separate lines

### Questions:
- [ ] Should the observation category be set by the profile for ECOG, staging, and other cases whe
    > Word ate Mark's sentence...I asked him to send again
- [ ] Should ECOG profile set the method as “Physical exam”?

### `example1-mcode-primary-cancer-condition`
- [x] You’ve got “left” in there three times, in the code, bodysite, and laterality. Since the site is precoordinated in the code, only the code is necessary.

### `example1-mcode-tumor-marker-pr`
- [x] valueCodeableConcept preferred for this LOINC code is Preferred Answer List LL2038-9, where Negative value is LA6577-6
    > I have changed to LOINC for now, but I will check with May/Carmela per the next item
- [ ] Consult with May and Carmela and decide whether the LOINC or SNOMED is preferred in this instance (also with other tumor marker tests).
- [ ] Note that the existing example mCODETumorMarkerExample01 for LOINC 39004-7 uses SNOMED for an answer, but that test does NOT have a preferred LOINC answer.
    > I have the same situation with `LNC#72382-5 "HER2 [Units/volume] in Tissue by Immunoassay"`, which does not have a preferred answer list. I have switched this to a LOINC answer for now for consistency with the other instances conforming to `TumorMarker` but I will discuss with May/Carmela.

### `example1-mcode-tumor-marker-21-gene-panel`
- [x] valueQuantity.unit = "ScoreOf" – Where does that come from?
    > This is a mistake, should have been `* valueQuantity = UCUM#{ScoreOf}`. Fixed.
- [x] Can you use a coded unit of measure?
- [x] Should this test have an interpretation?
    > Yes, good idea. I increased the score in the example from 37 to 47 to make it unambiguously "high" (per NCCN guidelines) and added the corresponding interpretation.
- [ ] (I'm adding this item for myself. --Max) Consult with May about her concerns with representing the results of the 21 gene panel using the `TumorMarker` profile.

### `example1-mcode-cancer-genomics-report`
- [x] Fill in the reporting lab information
- [x] Please confirm with May that you have the right approach here. I interpret your example as:
    - Seven different genes were studied
    - A variant in one gene (PALB2) was found
    > This is what I was trying to say. I will confirm with May that she has the same interpretation.
- [ ] Is it true that this implies negative for BRCA1, BRCA2, and the others? Shouldn’t this be more explicit?
    > This is the correct interpretation. I believe the absence of a variant for a given gene in a genetics report implies no significant findings.
    >
    > This does raise the question of which variants we would like to see in these reports. In addition to likely pathogenic/pathogenic (LP/P) variants I also think variants of unknown significance (VUS) should be included. The interpretation of VUS can change based on additional clinical evidence (which could come from aggregating data via mCODE-conforming resources), or from changes in ClinVar (which is also known to mis-classify variants as VUS in some cases). I'm not sure if there's a logical place to offer guidance on this for implementers, or if there's even an opportunity for implementers to determine which variants are included (it may just be whatever the genomics lab provides in many cases).
    >
    > I will discuss with May before checking this off in case I'm missing anything.

- [x] You don’t need to specify the code since it is inherited from the profile

### `MedicationStatements`
- [x] You didn’t capture the frequency of doses in structured data. It’s not required, but for demonstration maybe you should.
- [x] You could get ¼ style point for aliasing http://terminology.hl7.org/CodeSystem/medication-statement-category

### `example1-diagnosticreport-pathology`
- [x] Fill in the reporting lab information
    > Added in `performer` and `resultsInterpreter` references -- @markkramerus, is this what you're looking for?

### `example1-us-core-condition-anxiety`
- [x] In CodeableConcepts, folks generally don’t set the text if the Coding has a text. So you can cut the lines
    - `* clinicalStatus.text = "Active"`
    - `* verificationStatus.text = "Confirmed"`

    > Whoops, this was a bug in my JSON->FSH conversion script that I didn't catch.

### `example1-family-member-history-uncle`
- [x] The uncle was lucky that pancreatic cancer didn’t cause his death. Should you consider changing that?

masnick commented:
There is [an error](https://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/extended-example-1/qa.html#_scratch_ig-build-temp-2QTLD9_repo_input_examples_FamilyMemberHistory-example1-family-member-history-aunt) in the `FamilyMemberHistory` resources that I don't know how to fix:

> age-1: There SHALL be a code if there is a value and it SHALL be an expression of time. If system is present, it SHALL be UCUM. If value is present, it SHALL be positive. [(code.exists() or value.empty()) and (system.empty() or (system = %ucum)) and (value.empty() or value.hasValue().not() or (value > 0))]
